util/maven/generate_poms.py

```diff
@@ -58,6 +58,29 @@ def pom_deps(label):
         'name': 'Dagger',
         'artifact': 'dagger',
     },
+    '//gwt:gwt': {
+        'name': 'Dagger GWT',
+        'artifact': 'dagger-gwt',
+        'manual_dependencies': '''
```

Comment:
I'd rather do it as a follow-up if you don't mind, as I'm not quite sure how to do the "that one has a classifier, that one hasn't" dance and I fear we'd have to do a bit of back-and-forth.